src/data-formatting/brp.ts

```diff
@@ -137,6 +137,12 @@ export function getFullName(person: Persoon) {
   }`;
 }
 
+export function getFullAddress(address: Adres) {
+  return `${address.straatnaam} ${address.huisnummer} ${address.huisletter ||
+    address.huisnummertoevoeging ||
```

Comment:
wat als een huis een huisletter en toevoeging heeft? Kan dat?

Reply:
geen idee. Laten we dat even aan Reinder vragen